lib/dl_cache_engine/pyproject.toml

```diff
@@ -0,0 +1,43 @@
+
+[tool.poetry]
+name = "datalens-cache-engine"
+version = "0.0.1"
+description = ""
+authors = ["DataLens Team <datalens-opensource@yandex-team.ru>"]
+packages = [{include = "dl_cache_engine"}]
+license = "Apache 2.0"
+readme = "README.md"
+
+
+[tool.poetry.dependencies]
+attrs = ">=22.2.0"
+python = ">=3.10, <3.12"
+datalens-app-tools = {path = "../dl_app_tools"}
+datalens-constants = {path = "../dl_constants"}
+datalens-model-tools = {path = "../dl_model_tools"}
+lz4 = "==4.3.2"
```

Comment:
Looks like this https://github.com/datalens-tech/datalens-backend/pull/297/files#diff-28c03eb2fbe8fdad850bc5ad7cd54297926ba8a1e1136eb6a4b150a438b9bbdbR94 should be moved here

Reply:
done